go/oasis-test-runner/oasis/byzantine.go

```diff
@@ -74,8 +73,8 @@ func (worker *Byzantine) ModifyConfig() error {
 	worker.AddSeedNodesToConfig()
 
 	if !worker.entity.isDebugTestEntity {
-		dir := worker.entity.dir.String()
-		worker.Config.Registration.Entity = filepath.Join(dir, "entity.json")
+		entityID, _ := worker.entity.ID().MarshalText() // Cannot fail.
```

Comment:
The comment should say why this cannot fail (e.g. Returned err is always nil, MarshalText never fails, ....). Since the method returns an error, we could also return it here and then we don't need to care if the error is possible or not. Or maybe we refactor `MarshalText` to not return an error, just like cbor doesn't return an error when something is encoded.

Reply:
We cannot refactor as this is part of the Go standard text marshaller interface.